cluster-provision/k8s/1.17/provision.sh

```diff
@@ -9,8 +9,8 @@ xfs_growfs -d /
 
 cni_manifest="/tmp/cni.yaml"
 
-setenforce 0
-sed -i "s/^SELINUX=.*/SELINUX=permissive/" /etc/selinux/config
+setenforce 1
+sed -i "s/^SELINUX=.*/SELINUX=enforcing/" /etc/selinux/config
```

Comment:
Did you consider just removing these 2 lines instead, and let the OS keep its default behavior?

Reply:
I think we need to explicitly enforce selinux to rely on it.

Reply:
How about a test that breaks the script if "SELINUX=enforcing" doesn't grep in `/etc/selinux/config`? My concern is that this script gets used on some distro that doesn't support enforcing, silently enables it and breaks random stuff.
This is quite minor though, feel free to ignore.

Reply:
Actually I've been testing this locally and ran into the same timeout as the CI lane!
Looks like that script also applies to the VM that... builds the VM?
You can see SELinux errors if you VNC to 172.17.0.2 before the timeout.

Anyway, keeping `setenforce 0` and removing the `sed` line altogether fixes it for me. (i.e. the VM builds fine and the final image is set to enforcing)